### PR TITLE
feat: add default widget

### PR DESCRIPTION
**Pull request by pwltr**
### Description

- Widgets: Added the Bitcoin price widget (1D USD) as a default preinstalled widget on fresh installs.
- Backup: backup on delete widget
- rename slashtags2 -> slashtags

### Linked Issues/Tasks

https://github.com/synonymdev/bitkit/issues/1864

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [x] No test

### CODE REVIEW

**Comment by limpbrains** [comment]
initialSlashtagsState.profiles should be removed.
Also, please, rename src/utils/slashtags2

**Review by limpbrains — APPROVED** [review]

**Comment by limpbrains** [comment]
Any idea why e2e test keeps failing? Maybe try to workaround it?

File: src/screens/Wallets/index.tsx [hunk 1]
```diff
@@ -55,6 +59,7 @@ const Wallets = ({ navigation, onFocus }: Props): ReactElement => {
 	const widgets = useAppSelector(widgetsSelector);
 	const insets = useSafeAreaInsets();
 	const { t } = useTranslation('wallet');
+	const { config } = useSlashfeed({ url: PriceFeedURL });
```
Comment (pwltr) [inline src/screens/Wallets/index.tsx:62]: Detox issue is with this hook, not really sure how to work around it @limpbrains

File: src/store/utils/settings.ts [hunk 2]
```diff
@@ -48,7 +49,7 @@ export const wipeApp = async ({
 			});
 		}
 
-		if (restartApp) {
+		if (restartApp && !__E2E__) {
```
Comment (pwltr) [inline src/store/utils/settings.ts:52]: This seems to fix the test, I think this is just a precaution anyway, can we leave this out for E2E tests?

Comment (limpbrains) [reply src/store/utils/settings.ts:52]: I think we can

**Review by limpbrains — APPROVED** [review]

**Comment by catch-21** [comment]
Verified on 8de4fa74f41be7613c28bbcd36be93d50948aa74
BTC/USD price widget set as default and backup is stored when it is deleted.

**Pull request merged**